src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ActionPlanService.kt

```diff
@@ -77,14 +77,23 @@ class ActionPlanService(
     val submittedActionPlan = updateDraftActionPlanAsSubmitted(draftActionPlan, submittedByUser)
     actionPlanValidator.validateSubmittedActionPlan(submittedActionPlan)
 
-    actionPlanSessionsService.createUnscheduledSessionsForActionPlan(submittedActionPlan)
```

Comment:
if our condition remains "allow bookings when there's a submitted plan", we have to keep doing this after submission -- it doesn't sound like we can change this now

Reply:
ah it was my understanding that we couldn't book anything until approval. i must have misheard. can we get confirmation on this? if the whole action plan approval process is motivated by a billing concern, i wouldn't expect we should allow SPs to book appointments for any proposed sessions until approval by PP.

Reply:
confirmed: you're right -- given it's more complex for us to reduce booked sessions

**decision**: enable bookings of appointments once action plan is approved